dart/example_web/pubspec.yaml

```diff
@@ -8,9 +8,12 @@ environment:
 
 dependencies:
   sentry:
```

Comment:
should we do `any` here too or removing from `any` from `sentry_flutter`? just for the sake of equal configuration.

Reply:
Yea can do that to make it more clear